packages/synapse-types/src/UserProfile.ts

```diff
@@ -38,5 +38,6 @@ export type UserProfile = {
   notificationSettings?: Settings // Settings Contains a user's notification settings
   preferences?: UserPreference[]
   createdOn?: string // The date this profile was created
+  twoFactorAuthEnabled: boolean // Flag that indicates if the user has two factor authentication enabled.
```

Comment:
Should this be optional? I thought only ACT would see this for other users.

Reply:
Good call! It can be seen by ACT and Synapse admins for other users -- I'll make it optional